src/platform/windows.rs

```diff
@@ -65,6 +110,15 @@ impl<T> EventLoopBuilderExtWindows for EventLoopBuilder<T> {
         self.platform_specific.dpi_aware = dpi_aware;
         self
     }
+
+    #[inline]
+    fn with_translate_accel_callback(
+        &mut self,
+        callback: Box<dyn FnMut(winapi::um::winuser::MSG) -> bool>,
```

Comment:
As part of a public facing API `MSG` needs to be re-exported or even better a struct defined locally by `winit`.

Reply:
I guess reexporting is better so it can be used in other win32 apis directly.